src/connectors/query-rules/connectQueryRules.ts

```diff
@@ -175,7 +175,7 @@ const connectQueryRules: QueryRulesConnector = (render, unmount = noop) => {
       trackedFilters = {} as ParamTrackedFilters,
       transformRuleContexts = (rules => rules) as ParamTransformRuleContexts,
       transformItems = (items => items) as ParamTransformItems,
-    } = widgetParams || {};
+    } = widgetParams || ({} as QueryRulesConnectorParams);
```

Comment:
do we not have this pattern in more component?

Reply:
Only in `connectCurrentRefinements.ts`, but it was already correctly handled. There is no fallback in other components (some possible future improvements here).

Reply:
We could even use:

```ts
{} as typeof widgetParams
```

Reply:
Even better :+1: